src/linear_algebra/matrix/adjugate.lean

```diff
@@ -209,6 +222,20 @@ begin
     exact h ((symm_apply_eq σ).mp h') }
 end
 
+lemma adjugate_submatrix_equiv_self (e : n ≃ m) (A : matrix m m α) :
+  adjugate (A.submatrix e e) = (adjugate A).submatrix e e :=
```

Comment:
simp again?

Reply:
I'm not sure which way around is better, so took the safe option as not marking either direction simp. Do you have an argument one way or the other?

Reply:
I don't have a strong argument either way (we have both `adjugate_apply` and `submatrix_apply`), but at least one of them should be simp.

Reply:
I added it, since `transpose_submatrix` is simp.